config/configuration.json

```diff
@@ -107,6 +107,16 @@
       "tags":"highway"
     }
   },
+  "InvalidLanesTagCheck": {
+    "lanes.filter": "Lanes->1,1.5,2,3,4,5,6,7,8,9,10",
```

Comment:
What does it means if a the lane tag has a value of 1.5?

Reply:
Oh just found the answer here: https://wiki.openstreetmap.org/wiki/Key:lanes Which seems to indicate a narrow road. Interesting.